miden-lib/src/notes/mod.rs

```diff
@@ -50,6 +59,16 @@ pub fn create_note(
             ProgramAst::from_bytes(p2idr_bytes).map_err(NoteError::NoteDeserializationError)?,
             vec![target.into(), recall_height.into(), ZERO, ZERO],
         ),
+        Script::ASWAP {
+            faucet_id,
+            amount,
+            tag,
+            recipient,
+        } => (
+            ProgramAst::from_bytes(aswap_bytes).map_err(NoteError::NoteDeserializationError)?,
+            // vec![faucet_id.into(), amount.into(), tag, recipient[0]],
+            vec![faucet_id.into(), amount.into(), tag, recipient],
```

Comment:
Building inputs for this script would be a bit more involved. I'd probably do it in a separate helper function.

Reply:
Yes